tools/configen/configen/configen.py

```diff
@@ -128,15 +128,43 @@ def is_incompatible(type_: Type[Any]) -> bool:
     return True
 
 
+def get_default_flags(module: ModuleConf) -> List[Parameter]:
+
+    def_flags: List[Parameter] = []
+    if module.default_flags and hasattr(module.default_flags, "items"):
+        for name, default in module.default_flags.items():
+            if default:
+                flag_type = Flags.__annotations__[name].__args__[0].__name__
+                if "ConvertMode" in flag_type:
+                    flag_type = "str"
+                    default = str(default).strip("ConvertMode.").lower()
+
+                default = f'"{default}"' if flag_type == "str" else default
+
+                def_flags.append(
+                    Parameter(
+                        name=name,
+                        type_str=flag_type,
+                        default=default,
+                    )
+                )
+
```

Comment:
I was thinking something along the lines of :
```python
if module.default_flags._recursive_ is not None:
  def_flags.append(Parameter(name="_recursive_", type_str=bool, default=module.default_flags._recursive_ ))

if module.default_flags._convert_ is not None:
  def_flags.append(Parameter(name="_convert_", type_str=str, default=module.default_flags._convert_.name))
```

(I didn't run it so there may need to be some tweaks).

Reply:
done with tweaks